src/pam/oslogin_sshca.cc

```diff
@@ -212,21 +214,20 @@ static int GetExtension(pam_handle_t *pamh, const char *key, size_t k_len, char
 
   // Skip critical options.
   if (GetString(&key_b64, &n_len, NULL, NULL) < 0) {
-    PAM_SYSLOG(pamh, LOG_ERR, "Failed to skip cert's \"critical options\" "
-               "field.");
+    SysLogErr("Failed to skip cert's \"critical options\" field.");
     goto out;
   }
 
   // Get extensions buffer.
   if (GetString(&key_b64, &n_len, &tmp_exts, &tmp_exts_len) < 0) {
-    PAM_SYSLOG(pamh, LOG_ERR, "Failed to get cert's \"extensions\" field.");
+    SysLogErr("Failed to get cert's \"extensions\" field.");
     goto out;
   }
 
   // The field extensions is a self described/sized buffer.
   tmp_head = tmp_exts;
   if (GetString(&tmp_exts, &tmp_exts_len, exts, &ret) < 0) {
-    PAM_SYSLOG(pamh, LOG_ERR, "Failed to read google's extension.");
+    SysLogErr("Failed to read google's extension.");
```

Comment:
suggestion: "Google" instead of "google"

Reply:
Updated, thanks.